src/ensembl/src/content/app/custom-download/components/checkbox-grid/CheckboxGrid.test.tsx

```diff
@@ -49,57 +52,65 @@ describe('<CheckboxGrid />', () => {
     jest.resetAllMocks();
   });
 
-  let wrapper: any;
   const defaultProps = {
     options: defaultOptions,
     onChange,
     label: faker.lorem.word()
   };
+  const renderCheckboxGrid = (props?: Partial<CheckboxGridProps>) =>
+    render(<CheckboxGrid {...defaultProps} {...props} />);
 
   it('renders without error', () => {
-    wrapper = mount(<CheckboxGrid {...defaultProps} />);
-    expect(wrapper.find(CheckboxGrid).length).toEqual(1);
+    expect(() => {
+      renderCheckboxGrid();
+    }).not.toThrow();
   });
 
   it('renders N number of checkboxes based on the options', () => {
-    wrapper = mount(<CheckboxGrid {...defaultProps} />);
+    const { container } = renderCheckboxGrid();
 
-    expect(wrapper.find(Checkbox).length).toEqual(defaultOptions.length);
+    expect(container.querySelectorAll('.defaultCheckbox')?.length).toEqual(
+      defaultOptions.length
+    );
   });
 
   it('sorts the checkboxes based on the options array', () => {
-    wrapper = mount(<CheckboxGrid {...defaultProps} />);
-
-    const firstGridContainer = wrapper.find('.checkboxGridContainer').first();
+    const { container } = renderCheckboxGrid();
 
+    const firstGridContainer = container.querySelectorAll(
+      '.checkboxGridContainer'
+    )[0];
```

Comment:
that's what the `.querySelector` method does already — it returns the first match to the query

Reply:
I've updated it here and in other places